1.1/spec/11-Part-08.adoc

```diff
@@ -481,3 +481,109 @@ geof:getSRID (geom: ogc:geomLiteral): xsd:anyURI
 
 Returns the spatial reference system URI for `geom`.
 
+===== Function: geof:maxX
+
+```
+geof:maxX (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the maximum X coordinate for `geom`.
+
+===== Function: geof:maxY
+
+```
+geof:maxY (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the maximum Y coordinate for `geom`.
+
+===== Function: geof:maxZ
+
+```
+geof:maxZ (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the maximum Z coordinate for `geom`.
+
+===== Function: geof:minX
+
+```
+geof:minX (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the minimum X coordinate for `geom`.
+
+===== Function: geof:minY
+
+```
+geof:minY (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the minimum Y coordinate for `geom`.
+
+===== Function: geof:minZ
+
+```
+geof:minZ (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the minimum Z coordinate for `geom`.
+
+=== Spatial Aggregate Extension
+
+This clause establishes the Spatial Aggregate extension requirements class with URI `/req/spatial-aggregate-extension`.
+Spatial Aggregate Functions may be used in the SELECT used in the SELECT, HAVING and ORDER BY clauses of a SPARQL query.
+Such functions calculate an aggregate value over a group of solutions. Solution groups are determined by a GROUP BY clause. 
+All solutions form a single group if no GROUP BY is specified.
+
+==== Spatial Aggregate Functions
+
+===== Function: geosaf:BBOX
+
+```
+geosaf:BBOX (ogc:geomLiteral): ogc:geomLiteral
+```
+Calculates a minimum bounding box of the set of given geometries.
+
+===== Function: geosaf:BoundingCircle
+
+```
+geosaf:BoundingCircle (ogc:geomLiteral): ogc:geomLiteral
+```
+Calculates a minimum bounding circle of the set of given geometries.
+
+===== Function: geosaf:Centroid
+
+```
+geosaf:Centroid (ogc:geomLiteral): ogc:geomLiteral
+```
+Calculates the centroid of the set of given geometries.
+
+===== Function: geosaf:Concat_Lines
+
+```
+geosaf:Concat_Lines (ogc:geomLiteral): ogc:geomLiteral
```

Comment:
Can you please change this function name from `Concat_Lines` to `ConcatLines` for consistent styling?